Gemfile

```diff
@@ -12,6 +12,7 @@ gem "pg", "~> 1.1"
 gem "puma", "~> 5.0"
 gem "sass-rails", ">= 6"
 gem "view_component", "~> 2.42"
+gem "classy-yaml", "~> 0.5"
```

Comment:
Please move this gem to the bottom group and sort alphabetically. I'm trying to keep this area for Rails and core gems.

Reply:
Done!